Challenge2/app/src/main/java/com/bumble/challenge2/navmodel/custombackstack/operation/ToggleFreeze.kt

```diff
@@ -10,14 +10,20 @@ import kotlinx.parcelize.Parcelize
 class ToggleFreeze<T : Any> : CustomBackStackOperation<T> {
 
     override fun isApplicable(elements: NavElements<T, CustomBackStack.State>): Boolean =
-        elements.any { TODO("(1) Define applicability logic") }
+        elements.any {
```

Comment:
In this case we'd like to be able to apply the `ToggleFreeze` operation when an element is Active or Frozen.
`elements.any { it.targetState is Active || it.targetState is Frozen }`